### PR TITLE
fix(crypto): htlc lock buffer allocation

### PR DESCRIPTION
**Pull request by KovacZan**
Changing the size of bytebuffer in htlc serialization, because is larger then needed

## What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactor
- [ ] Performance
- [ ] Tests
- [ ] Build
- [ ] Documentation
- [ ] Code style update
- [ ] Continuous Integration
- [ ] Other, please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Does this PR release a new version?

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `develop` branch, _not_ the `master` branch
- [x] All tests are passing
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

### CODE REVIEW

File: packages/crypto/src/transactions/types/htlc-lock.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export class HtlcLockTransaction extends Transaction {
     public serialize(options?: ISerializeOptions): ByteBuffer {
         const { data } = this;
 
-        const buffer: ByteBuffer = new ByteBuffer(8 + 32 + 4 + 8 + 21, true);
+        const buffer: ByteBuffer = new ByteBuffer(8 + 32 + 1 + 8 + 21, true);
```
Comment (dated) [inline packages/crypto/src/transactions/types/htlc-lock.ts]: Isn't the expiration value only 4 bytes?

```suggestion
        const buffer: ByteBuffer = new ByteBuffer(8 + 32 + 1 + 4 + 21, true);
```

Comment (supaiku0) [reply packages/crypto/src/transactions/types/htlc-lock.ts]: Yes, good catch.

Comment (dated) [reply packages/crypto/src/transactions/types/htlc-lock.ts]: The same goes for https://github.com/ArkEcosystem/java-crypto/pull/56 then.

Comment (KovacZan) [reply packages/crypto/src/transactions/types/htlc-lock.ts]: nice, thanks

**Pull request merged**